freestyle-twitter-future/src/main/scala/future.scala

```diff
@@ -0,0 +1,37 @@
+package freestyle
+package twitter
+
+import com.twitter.util.Future
+import cats.MonadError
+
+class FreestyleTwitterMonad extends MonadError[Future, Throwable] with Capture[Future] {
+
```

Comment:
I removed it because 'ap' was never called. I'm out of a computer today but if you think it makes sense to add it feel free to commit on this branch and we'll get it merged